paper/basic_training.tex

```diff
@@ -616,28 +620,52 @@ \subsubsection{Popular Thermostats}
 
 \subparagraph{Deterministic Thermostats}
 \begin{enumerate}[listparindent=\parindent]
-    \item \textbf{Velocity Rescaling}
+    \item \textbf{Simple Velocity Rescaling}
 
-        The velocity rescaling thermostat is one of the simplest thermostats to implement, however, this thermostat is also one of the most non-physical thermostats as well.
+        The simple velocity rescaling thermostat is one of the easiest thermostats to implement, however, this thermostat is also one of the most non-physical thermostats as well.
 
          This thermostat relies on rescaling the momenta of the particles every $N$ timesteps based on their ratio of the instantaneous temperature to the target temperature\cite{thermostatAlgorithms2005}. 
          This leads to multiple issues when trying to sample data using this thermostat.
          First, this method is not reversible, there lacks a way for the particles to have knowledge of their previous thermal history.
          This makes any dynamical value impossible to measure (diffusion for example). 
          There is also no distribution of momenta; this thermostat does not sample the canonical distribution and cannot be used for sampling at thermal equilibrium.
+         The isokinetic (constant kinetic energy) ensemble samples the
+         same configurational phase space as the canonical ensemble, so
+         position-dependent equilibrium properties can be obtained
+         equivalently with either
+         ensemble\cite{Minary:2003:JChemPhys:Algorithms}.  However, the
+         isokinetic ensemble is not properly sampled by the simple
+         velocity rescaling thermostat, and its usage can lead to
+         simulation artifacts, so it is not
+         recommended\cite{Braun:2018:arXiv:Anomalous}.
 
     \item \textbf{Hoover (Gaussian)}
+        \todo[inline, color={green!20}]{EB: I recommend referring to
+        this thermostat as the Gaussian Thermostat. I've seen it
+        referred to as the Hoover-Evans thermostat as well (I
+        haven't seen it called the Hoover thermostat, but I'm sure
+        somebody has called it that), but simply calling it the
+        Gaussian thermostat avoids
+        wading into the argument about who came up with it
+        first.}
         
-        This thermostat is also an example of a velocity rescaling thermostat; however, the goal of the Gaussian thermostat is to ensure the change in
-        temperature $\Delta T$ is always 0 ($\Delta T \equiv 0$).
+        The goal of the Gaussian thermostat is to ensure the change in
+        instantaneous temperature $\Delta T$ is always 0 ($\Delta T \equiv 0$), which
+        is accomplished by modifying the force calculation with the form
+        $F = F_{interaction} + F_{Lagrange multiplier}$, where
+        $F_{interaction}$ is the standard interactions calculated during
+        the course of the simulation and $F_{Lagrange multiplier}$ keeps
```

Comment:
I think this space in the subscript wouldn't show up properly?

Reply:
Right, fixed.